app.js

```diff
@@ -3,20 +3,53 @@ var request = require("request");
 var app = express();
 var port = process.env.port || 3000;
 
+require('dotenv').config();
+
 app.set("view engine", "ejs");
 app.use(express.static("public"));
 
 app.get("/", function(req, res) {
-    request("https://v2.jokeapi.dev/joke/Any", function(err, response, body) {
+    request(
+        {method: 'GET',
+        url: 'https://dad-jokes.p.rapidapi.com/random/joke',
+        headers: {
+          'x-rapidapi-host': 'dad-jokes.p.rapidapi.com',
+          'x-rapidapi-key': process.env.Jokes_API_KEY,
+          useQueryString: true
+        }
+        }, function(err, response, body) {
         if(err) {
             console.log(err);
         } else {
-            data = JSON.parse(body);
+            data = {
+                body:JSON.parse(body).body[0],
+                urlName:"dadJokes"
+                };
+            res.render("getJokes",{data: data});
+        }
+    })
+});
+app.get("/chuck", function(req, res) {
```

Comment:
Making a separate endpoint to go to get different jokes is not ideal. If we add 5-6 genres, we will end up with 5-6 endpoints all doing the same thing(calling an API and showing results). Make all the changes in the same endpoint and use functions and conditions to show the result.

One way is when a genre is selected by the user(user clicked Chuck Norris joke button on the home page, we can use the button to send input back to the nodejs app to understand which genre user wants and then show them the jokes based on that.